styles/components/_trading-card-image.scss

```diff
@@ -0,0 +1,154 @@
+.cmp-trading-card-image {
+  position: relative;
+  width: fit-content;
+
+  &__overlay {
+    position: relative;
+    height: 350px;
+    z-index: 30;
+
+    &::after {
+      position: absolute;
+      left: 0;
+      top: 0;
+      right: 0;
+      bottom: 0;
+      border-radius: 50%;
+      content: "";
+      transition: background-color 125ms linear;
+    }
+
+    &--gray::after {
+      background-image:
+        linear-gradient(
+          90deg,
+          color("gray-200") -15%,
+          transparent 39%
+        );
+    }
+
+    &--blue::after {
+      background-image:
+        linear-gradient(
+          90deg,
+          color("blue-50-transparency-75") -15%,
+          color("blue-50-transparency-50") 20%,
+          transparent 66%
+        );
+    }
+
+    &--green::after {
+      background-image:
+        linear-gradient(
+          90deg,
+          color("green-50") 20%,
+          color("green-50-transparency-50") 81%,
+          transparent 172%
+        );
+    }
+  }
+
+  &__image {
+    position: relative;
+    height: 350px;
+    width: 350px;
+    border-radius: 50%;
+    z-index: 50;
+
+    &::after {
+      position: absolute;
+      left: 0;
+      top: 0;
+      right: 0;
+      bottom: 0;
+      border-radius: 50%;
+      content: "";
+    }
+
+    &--blue::after {
+      background-color: color("blue-100-transparency-50");
+    }
+
+    &--green::after {
+      background-color: color("green-100-transparency-50");
+    }
+  }
+
+  &__hours-container {
+    position: absolute;
+    right: 0;
+    bottom: space("lg");
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    width: 6rem;
+    height: 6rem;
+    border-radius: 50%;
+    font-weight: 700;
+    color: color("black");
+    transition: background-color 125ms linear;
+    z-index: 70;
+
+    &--gray {
+      background-color: color("gray-100");
+    }
+
+    &--blue {
+      background-color: color("blue-50");
+    }
+
+    &--green {
+      background-color: color("green-50");
+    }
+  }
+
+  &__hours {
+    font-size: font-size("xl");
+  }
+}
+
+.cmp-trading-card-image:hover {
```

Comment:
Something feels off about the hover states, but I can't put my finger on it. Might be the transition or the linear gradient... I'd pair with a designer and see if they have any ideas.

Reply:
Current gradient and hover state is much closer to the design than what it was before, but now we're without the filter over the image itself. I'll be pairing with a designer to try to rectify that!